object_manager.go

```diff
@@ -62,6 +64,8 @@ type IBObjectManager interface {
 	GetARecord(dnsview string, recordName string, ipAddr string) (*RecordA, error)
 	GetAAAARecord(dnsview string, recordName string, ipAddr string) (*RecordAAAA, error)
 	GetAAAARecordByRef(ref string) (*RecordAAAA, error)
+	GetAliasRecordByRef(ref string) (*RecordAlias, error)
+	GetAllAliasRecord(queryParams *QueryParams) ([]RecordAlias, error)
```

Comment:
As func **GetAllAliasRecord** filters data based on queryParams refactor the func name to **GetAliasRecords**